data/json/recipes/other/other.json

```diff
@@ -15,8 +15,7 @@
         [ "fish", 1 ],
         [ "dry_meat", 1 ],
         [ "dry_fish", 1 ],
-        [ "corn", 1 ],
-        [ "irradiated_corn", 1 ],
```

Comment:
No irradiated?

Reply:
from what I can gather irradiated corn is the full ear, not just the grain

Reply:
So that would be irradiated_corn_cob?